PrebidMobile/PrebidMobile-core/src/main/java/org/prebid/mobile/rendering/sdk/StatusRequester.java

```diff
@@ -1,37 +1,37 @@
 package org.prebid.mobile.rendering.sdk;
 
-import androidx.annotation.NonNull;
-import androidx.annotation.Nullable;
 import org.json.JSONException;
 import org.json.JSONObject;
 import org.prebid.mobile.LogUtil;
 import org.prebid.mobile.PrebidMobile;
-import org.prebid.mobile.api.exceptions.InitError;
-import org.prebid.mobile.rendering.listeners.SdkInitializationListener;
 import org.prebid.mobile.rendering.networking.BaseNetworkTask;
 import org.prebid.mobile.rendering.networking.ResponseHandler;
 import org.prebid.mobile.rendering.networking.tracking.ServerConnection;
 
+/**
+ * It checks if Prebid server is accessible. It makes request to `/status` endpoint.
+ * For now, it supports only a server host that ends with `/openrtb2/auction`.
+ */
 public class StatusRequester {
 
     private static final String TAG = StatusRequester.class.getSimpleName();
 
-    public static void makeRequest(@Nullable SdkInitializationListener listener) {
+    public static void makeRequest() {
         String url = PrebidMobile.getPrebidServerHost().getHostUrl();
         if (url.contains("/openrtb2/auction")) {
             String statusUrl = url.replace("/openrtb2/auction", "/status");
             ServerConnection.fireWithResult(
                 statusUrl,
-                getResponseHandler(listener)
+                getResponseHandler()
             );
         } else if (url.isEmpty()) {
-            onInitError("Please set host url (PrebidMobile.setPrebidServerHost) and only then run SDK initialization.", listener);
+            LogUtil.error(TAG, "Please set host url (PrebidMobile.setPrebidServerHost) and only then run SDK initialization.");
```

Comment:
it's not an error. We just can't recognize the given URL. We shouldn't log it as an error.